### PR TITLE
7852 scalar price fixes

### PR DESCRIPTION
**Pull request by tinybike**
Resolves:
https://app.clubhouse.io/augur/story/7852/scalar-order-creation-transactions-fail
https://app.clubhouse.io/augur/story/7863/speed-up-canned-markets-scripts-called-from-dp
https://app.clubhouse.io/augur/story/7864/make-polling-block-interval-an-adjustable-parameter-in-augur-js

### CODE REVIEW

File: test/unit/trading/calculate-trade-cost.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+describe("trading/calculate-trade-cost", function () {
+  var test = function (t) {
+    it(t.description, function () {
+      t.assertions(calculateTradeCost(t.params));
+    });
+  };
+  test({
+    description: "scalar bid",
+    params: {
+      displayPrice: "20",
```
Comment (MicahZoltu) [inline test/unit/trading/calculate-trade-cost.js]: Recommend one or more test cases with a negative `displayPrice`.

File: scripts/set-local-dev-env.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # All of these keys are optional, and have the defaults listed here
-export USE_NORMAL_TIME="false"                # need to beable to change time manually
+# export USE_NORMAL_TIME="false"                # need to beable to change time manually
```
Comment (nuevoalex) [inline scripts/set-local-dev-env.sh]: why remove this?

Comment (tinybike) [reply scripts/set-local-dev-env.sh]: I forgot to uncomment it D:
Will fix

File: test/unit/trading/calculate-trade-cost.js (hunk 1 above)
Comment (tinybike) [reply test/unit/trading/calculate-trade-cost.js]: Good thinking!  Do you want to write a couple?  (You know you do)
